authors/apps/authentication/backends.py

```diff
@@ -1,41 +1,49 @@
 from django.conf import settings
+
```

Comment:
please remove white spaces on lines 2, 4 and 11

Reply:
@sanya-kenneth Thanks for the feedback but those lines in my imports are necessary to differentiate the level of imports.